Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM debian:stable-slim
+SHELL [ "/bin/sh", "-ec" ]
 
-RUN apt-get update && apt-get -uy upgrade
-RUN apt-get -y install ca-certificates && update-ca-certificates
+ENV DEBCONF_NONINTERACTIVE_SEEN=true DEBIAN_FRONTEND=noninteractive DEBIAN_PRIORITY=critical TERM=linux
```

Comment:
Why are these required?

Reply:
These environment variables reduce amount of (useless) information in build logs, mostly debconf tuning.